### ISSUE TITLE
Configuration Options

### ISSUE THREAD

**Issue by qvalentin** [mrjosh/helm-ls#47]
The lsp should implement configuration options as defined by the protocol: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_configuration

Possible options:
- filename(s) of values.yaml

**Comment by qvalentin** [comment]
Basic feature will be added with https://github.com/mrjosh/helm-ls/pull/32

**Comment by yvesjans** [comment]
Is this already possible to configure/use? That would be awesome as we're using separate value files for environments, e.g.
values.yaml: shared config
values-nonprod.yaml: nonprod config
values-production.yaml: production config

**Comment by qvalentin** [comment]
Fixed in [v0.0.10](https://github.com/mrjosh/helm-ls/releases/tag/) with these defaults:

```lua
    valuesFiles = {
      mainValuesFile = "values.yaml",
      lintOverlayValuesFile = "values.lint.yaml",
      additionalValuesFilesGlobPattern = "values*.yaml"
    },
```
